romancal/dq_init/dq_init_step.py

```diff
@@ -83,6 +83,21 @@ def process(self, input):
                 **reference_file_models,
             )
 
+            # copy original border reference file arrays (data and dq)
+            # to their own attributes. they will also remain attached to
+            # the science data until they are trimmed at ramp_fit
+            # these arrays include the overlap regions in the corners
+
+            output_model.border_ref_pix_right = output_model.data[:, -4:]
+            output_model.border_ref_pix_left = output_model.data[:, :4]
+            output_model.border_ref_pix_top = output_model.data[:4, :]
+            output_model.border_ref_pix_bottom = output_model.data[-4:, :]
```

Comment:
These need to go back to being 3D.. otherwise you get output objects like:
  border_ref_pix_top: !core/ndarray-1.0.0
    shape: [4, 4096, 4096]
    
When it should be number_of_groups, 4 , 4096